content/id/includes/task-tutorial-prereqs.md

```diff
@@ -1,8 +1,8 @@
-Kamu harus memiliki klaster Kubernetes, dan perangkat baris perintah `kubectl` 
-juga harus dikonfigurasikan untuk berkomunikasi dengan klaster kamu. Jika kamu 
-belum punya klaster, kamu dapat membuatnya dengan menggunakan
-[Minikube](/docs/setup/learning-environment/minikube/),
-atau kamu dapat menggunakan salah satu tempat bermain Kubernetes ini:
+Kamu harus memiliki klaster Kubernetes, dan perangkat baris perintah kubectl
+juga harus dikonfigurasikan untuk berkomunikasi dengan klastermu. Jika kamu
+belum memiliki klaster, kamu dapat membuatnya dengan menggunakan
+[minikube](/id/docs/tasks/tools/#minikube),
```

Comment:
FYI `/id/docs/tasks/tools/` also need to be updated, maybe later on separate PR.

Reply:
@habibrosyad  I think minikube should be redirected into official web page.

Reply:
In the upstream EN they do not link minikube directly into its official web https://raw.githubusercontent.com/kubernetes/website/master/content/en/includes/task-tutorial-prereqs.md. However the [page](https://raw.githubusercontent.com/kubernetes/website/master/content/en/docs/tasks/tools/_index.md) (`/en/docs/tasks/tools/`) they linked, has a brief intro to minikube along with the direct link to the official web page. wdyt?

**Update**
I think we don't need to update `/id/docs/tasks/tools/`, it's pretty much in sync with `/en/docs/tasks/tools/`. I forgot that it is the page that @ariscahyadi [updated recently](https://github.com/kubernetes/website/commit/fbdd3880057472f36d19d8d9ea9514a631701e49#diff-d563f55a0b14e03f03184090f0b1e4576b5e6693085768be8f7c0b0e85ed0cf5).

Reply:
Yes, @habibrosyad... my change has been merged here https://github.com/kubernetes/website/blob/master/content/id/docs/tasks/tools/_index.md